modules/logging-feature-reference-5.6.adoc

```diff
@@ -89,8 +89,8 @@ include::snippets/logging-outputs-5.6-snip.adoc[]
 | Global proxy support  | &#10003; | &#10003;
 | x86 support           | &#10003; | &#10003;
 | ARM support           | &#10003; | &#10003;
-| PowerPC support       | &#10003; | &#10003;
-| IBM Z support         | &#10003; | &#10003;
+| {ibmpowerProductName} support       | &#10003; | &#10003;
```

Comment:
This module doesn't state where it is.

Reply:
Nice catch. No assemblies include this module.

Reply:
@libander Seems like you added that file just a month ago. Is it included anywhere? And what's the difference to modules/cluster-logging-feature-reference.adoc ?

Reply:
I think we can keep the change and mark this conversation as resolved.